tests/integration/suite/test_pod_security_policies.py

```diff
@@ -242,3 +245,70 @@ def test_pspt_binding(admin_mc, admin_pc, remove_resource):
         remove_resource(b)
     assert e.value.error.status == 422
     assert e.value.error.message == 'podSecurityPolicyTemplate not found'
+
+
+def test_project_action_set_pspt(admin_mc, admin_pc, remove_resource):
```

Comment:
This test and the other one changing the psp need to be marked as non-parallel. If they run at the same time the flag could be flipped by the other test.